.github/workflows/robustness.yaml

```diff
@@ -10,3 +10,4 @@ jobs:
       count: 12
       testTimeout: 30m
       artifactName: main
+      runs-on: "['ubuntu-latest-8-cores']"
```

Comment:
Can you extend this to robustness nightly running on amd64?

Reply:
Yes. I want to test it more time and then I will update it to nightly run as well.